src/store/RootStore/index.ts

```diff
@@ -0,0 +1,5 @@
+import RootStore from "./RootStore";
+
+export { default } from "./instance";
+
+export { RootStore };
```

Comment:
вроде вот так сократить можно
```suggestion
export { default as RootStore} from "./RootStore";

export { default } from "./instance";
```

некрит